components/ui/AppPageHeader.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <header
+    class="app-page-header"
+    :class="{ 'app-page-header_bg-grid': bgGrid }"
+  >
+    <div
+      class="app-page-header__container"
+      :class="{
+        'app-page-header__container_has-aside': hasAside,
+        'app-page-header__container_fixed-height': fixedHeight,
+      }"
+    >
+      <div class="app-page-header__top-link">
```

Comment:
What is top link?

Reply:
Same question

Reply:
Sorry for the lack of context here.

_Top Link_ would be an optional element at the top of the header, as you can see in the following image:

![image](https://user-images.githubusercontent.com/22047320/110473794-37f81680-80df-11eb-8d4d-2bac753f2bea.png)

Since its usage may be introduced only within the Textbook Demo for the moment, I'm just reserving some space for it. 

That space also serves as a consistent top _margin_ for all headers.